meerk40t/gui/wxmmain.py

```diff
@@ -1783,14 +1813,6 @@ def load(self, pathname):
                 self.set_file_as_recently_used(pathname)
                 if n != self.context.elements.note and self.context.auto_note:
                     self.context("window open Notes\n")  # open/not toggle.
-                try:
-                    if self.context.uniform_svg and pathname.lower().endswith("svg"):
-                        # or (len(elements) > 0 and "meerK40t" in elements[0].values):
-                        # TODO: Disabled uniform_svg, no longer detecting namespace.
-                        self.working_file = pathname
-                        self.validate_save()
-                except AttributeError:
-                    pass
```

Comment:
I moved this because it doesn't make sense to set the working file if we're just importing a file into the current project.